stateMutators.js

```diff
@@ -0,0 +1,212 @@
+const gameState = {
+    basket: {
+        basketValue: 0,
+        xPos: 50,
+        yPos: 10,
+        width: 30,
+        height: 10
+    },
+    score: 0,
+    livesRemaining: 3,
+    gameLevel: 1,
+    gameMode: 'running',
+    fallingObjects: [],
+}
+
+const setBasketPos = (state, basket, value) => {
```

Comment:
The `basket` param should be removed from this and the next few mutators. You can access the current state via `state.basket`

Reply:
Thanks for pointing those out, I missed them. Updated now